### PR TITLE
users can only manipulate own profile and articles

### PR DESCRIPTION
**Pull request by LeeRenJie**
depends on #35

### CODE REVIEW

File: alpha-blog/app/controllers/application_controller.rb [hunk 1]
```diff
@@ -10,5 +10,12 @@ def logged_in?
     !!current_user
   end
 
+  def require_user
+    if !logged_in?
```
Comment (tlkiong) [inline alpha-blog/app/controllers/application_controller.rb:14]:
```
unless logged_in?
```

**Review by tlkiong — CHANGES_REQUESTED** [review]

File: alpha-blog/app/controllers/application_controller.rb (hunk 1 above)
Comment (LeeRenJie) [reply alpha-blog/app/controllers/application_controller.rb:14]: `unless`  is better?

Comment (LeeRenJie) [reply alpha-blog/app/controllers/application_controller.rb:14]: image
How about this?
Better to do `unless current_user == @article.user` ?

Comment (tlkiong) [reply alpha-blog/app/controllers/application_controller.rb]: Personally, I prefer using `if`. But I bring this up here is to let you be aware of the existence of `unless` 
Your choice.

**Review by tlkiong — APPROVED** [review]

**Pull request merged**